Add policy for service app to read configuration (#963)
vtexdocs/dev-portal-content: docs/vtex-io/Getting-Started/back-end-development-guides/vtex-io-documentation-developing-service-configuration-apps.md

</details>

### Step 3 - Declaring policies and permissions

> You have to declare the required [policy](https://developers.vtex.com/docs/guides/vtex-io-documentation-policies) that gives permission to the service app to read the data from the configuration app. In the `manifest.json` file, add the following policy in the `"policies"` list:

Reviewer: ```suggestion
In the `manifest.json` file, add the `read-workspace-apps` [policy](https://developers.vtex.com/docs/guides/vtex-io-documentation-policies) to the `"policies"` list. This policy is responsible for allowing the service app to read the data from the configuration app.
```

Reviewer: como é um guia, quanto mais direto, melhor
